### ISSUE TITLE
Feature idea: add custom header images for redirects

### ISSUE THREAD

**Issue by SeanKilleen** [excellalabs/xluhco#96]
Sometimes when you post a link to social media, the header image reflects what the base page supplies, which isn't always ideal.

It would be interesting if xluh.co allowed a custom image to be presented so that when someone posted an http://xluh.co link to social media, it displayed the image of choice.

**Comment by SeanKilleen** [comment]
(for my own memory: this is per a Slack conversation with Shannon Gilhuley on 3/26/18)

**Comment by l-g-dev** [comment]
The way I envision this work is to read an optional 3rd column from the .csv file. This column will contain the URL of the custom image.

If this URL is present, the intermediate redirect page will display it, at the top of the page, on the original size.

Is this a good approach / do you have any suggestions?

**Comment by SeanKilleen** [comment]
@l-g-dev yes -- sort of. I think that you may want to look at where twitter, LinkedIn, and others display that image from -- it may be that we can include a URL without including an actual image in the page itself (for load times, etc.) .

Links to check out: 

* https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image.html
* https://www.addtoany.com/ext/meta-tags/

**Comment by l-g-dev** [comment]
I checked those links. Twitter displays the page on the same way I'm thinking to do, by loading the image from an external URL. I can't think of any different way to achieve this. Loading an image won't affect the speed of the page load, since it happens in parallel, but it is possible that the redirect happens before the image has fully loaded.

The only difference is that twitter doesn't read the external URL from a list. Instead it scrapes the HTML of the site provided, looking for a meta tag that indicates the URL of the desired image. We could implement a similar system, but it should be in a separate issue.

**Comment by l-g-dev** [comment]
Ok, that makes sense. I thought you wanted to actually display the image on redirect.cshtml, but I can see that's not what's originally requested. Just to display these images on the social media posts, I agree that the meta tags will suffice.
